applications/luci-app-acme/htdocs/luci-static/resources/view/acme.js

```diff
@@ -78,6 +78,18 @@ return view.extend({
 				"The first name will be the subject name, subsequent names will be alt names. " +
 				"Note that all domain names must point at the router in the global DNS."));
 		o.datatype = "list(string)";
+		o.validate = function (section_id, value) {
+			if (!/^[*a-z0-9.-]*$/.test(value)) {
```

Comment:
Should probably include upper-case letters as well? They don't really matter, but seems a bit strict to force people to lower-case everything?